awx/templates/rest_framework/api.html

```diff
@@ -39,7 +39,7 @@
           {% else %}
           <li><a href="{% url 'api:login' %}?next={{ request.get_full_path }}" data-toggle="tooltip" data-placement="bottom" data-delay="1000" title="Log in"><span class="glyphicon glyphicon-log-in"></span>Log in</a></li>
           {% endif %}
-          <li><a href="//docs.ansible.com/ansible-tower/{{short_tower_version}}/html/towerapi/index.html" target="_blank" data-toggle="tooltip" data-placement="bottom" data-delay="1000" title="{% trans 'API Guide' %}"><span class="glyphicon glyphicon-question-sign"></span><span class="visible-xs-inline">{% trans 'API Guide' %}</span></a></li>
+          <li><a href="//docs.ansible.com/automation-controller/latest/html/towerapi/" target="_blank" data-toggle="tooltip" data-placement="bottom" data-delay="1000" title="{% trans 'API Guide' %}"><span class="glyphicon glyphicon-question-sign"></span><span class="visible-xs-inline">{% trans 'API Guide' %}</span></a></li>
```

Comment:
Since this version of the API is AWX, let's have it point to: https://ansible.readthedocs.io/projects/awx/en/latest/rest_api/index.html

Reply:
maybe we can point to the api reference list https://ansible.readthedocs.io/projects/awx/en/latest/rest_api/api_ref.html

does this file get rewritten downstream to point to controller docs?

Reply:
@fosterseth, it gets pushed out to https://developers.redhat.com/api-catalog/api/ansible-automation-controller?extIdCarryOver=true&sc_cid=701f2000001Css5AAC.
Why not just send them to the entire API guide with instructions on how to search, sort, etc, not just only the generated reference?